index.js

```diff
@@ -123,6 +115,53 @@ function authorize () {
   }
 }
 
+/**
+ * Lookup the read conditions for a model
+ *
+ * @param  {Object} model  - class/entity instance
+ * @param  {Object} target - target instance
+ * @return {Object}
+ */
+
+function conditions (model, target) {
+  var config = modelConfig(model);
+
+  // no configuration found for
+  // the current model, quit
+  if (!config) {
+    return false;
+  }
+
+  // configure rules for
+  // this entity instance
+  var ability = new Ability();
+  config.call(ability, model);
+
+  // filter for relevant rules
+  var rules = ability.rules.filter(function (rule) {
+    return actionMatches('read', rule) &&
```

Comment:
Action name should be passed as an argument to `conditions`. When I built this library, I did not want to force any rule naming on developers and let them handle it. So it would be great if rule name was easily changeable.